tatt/tool.py

```diff
@@ -17,3 +19,21 @@ def idfun(x): return x
             seen[marker] = 1
             result.append(item)
     return result
+
+def get_repo_dir(repodir):
+    # Prefer the repo dir in the config
+    if repodir:
+        if os.path.isdir(repodir):
+            return repodir
+        else:
+            raise ValueError("Repo dir does not seem to be a directory")
+
+    # No path given in config
+    if os.path.isdir("/var/db/repos/gentoo/"):
+        print("Using /var/db/repos/gentoo/ as fallback")
+        return "/var/db/repos/gentoo"
+    elif os.path.isdir("/usr/portage/"):
+        print("Using /var/portage/ as fallback")
```

Comment:
no, "/usr/portage", not "/var/portage". Same in the commit message

Reply:
is he drunk-coding again?! :rofl: ..

Reply:
Just a typo..